database/metadata/databases/default/tables/public_people_list_view.yaml

```diff
@@ -20,6 +20,15 @@ object_relationships:
         remote_table:
           name: drvr_ethncty
           schema: lookups
+  - name: ems_pcr
```

Comment:
new object relationship between `people_list_view` and `ems__incidents` so that when we query people on the incident details page we know if they have already been linked so users wont be able to try to link them again